File: src/SomaConjNum.java
import java.util.Scanner;

public class SomaConjNum {
    public static void main(String[] args) {
        soma();
    }

    public static void soma() {
        Scanner prompt = new Scanner(System.in);
        System.out.println("Quantidade de números:");
        short n = prompt.nextShort();
        long soma = 0;

        for (short i = 0; i < n; i++) {
            System.out.println("Número:");
            int numero = prompt.nextInt();
            soma += numero;
        }

        System.out.println("Soma = " + soma);
        prompt.close();
    }
}

File: src/SomaConjNum.js
const prompt = require('prompt-sync')({ sigint: true });

function soma() {
  const n = parseInt(prompt("Quantidade de números:"));
  let soma = 0;

  for (let i = 0; i < n; i++) {
    const numero = parseInt(prompt("Número:"));
    soma += numero;
  }

  console.log(`Soma = ${soma}`);
}

soma();